components/renku_data_services/session/api.spec.yaml

```diff
@@ -111,93 +219,218 @@ paths:
           description: The session was removed or did not exist in the first place
         default:
           $ref: "#/components/responses/Error"
+      tags:
+        - session_launchers
+  /session_launchers/{launcher_id}/start:
```

Comment:
`/start` isn't needed since we `POST` to this endpoint.

Reply:
OK. This endpoint is not implemented here, so I will update the API spec in the next PR (https://github.com/SwissDataScienceCenter/renku-data-services/pull/127).